build:
  id: "KILO7_BACKEND_FINAL"

mqtt:
  host: "kilo-dev.local"
  port: 1883
  keepalive_s: 20
  ignore_drive_from_mqtt: true
  alert_min_interval_s: 2.0

control:
  cmd_ttl_ms: 250
  hb_ttl_ms: 1000

safety:
  override_required: true
  allow_clear_while_override: false
  imu_ttl_ms: 2000
  rollover_tilt_deg: 45
  rollover_hysteresis_deg: 5
  impact_accel_g_threshold: 2.0
  impact_hysteresis_g: 0.3
  perception_enforcement:
    enabled: false
    min_hazard_level: "CAUTION"
    min_stop_buffer_m: 0.0
    stale_denies: true

hardware:
  relay_mode: "rpi_gpio"
  relay_gpio_pin: 17
  relay_kill_level_low: false

safety_model:
  profile: "normal"
  profiles:
    crawl:
      reaction_time_s: 0.8
      brake_decel_mps2: 1.5
      buffer_m: 0.7
      max_speed_mps: 0.5
    normal:
      reaction_time_s: 0.6
      brake_decel_mps2: 2.0
      buffer_m: 0.5
      max_speed_mps: 1.2
    sport:
      reaction_time_s: 0.4
      brake_decel_mps2: 3.0
      buffer_m: 0.3
      max_speed_mps: 2.0

pwm:
  mode: "pigpio"
  steer_gpio_pin: 12
  throttle_gpio_pin: 13
  steer_center_us: 1500
  steer_span_us: 500
  throttle_center_us: 1500
  throttle_span_us: 500
